website/src/components/Schema/index.js

```diff
@@ -93,18 +93,18 @@ const Schema = ({ schema, menuItem, isLatestSchema }) => {
 
   const [currentShowingScript, setCurrentShowingScript] = React.useState(null);
```

Comment:
maybe update this to `currentShowingScripts` and `setCurrentShowingScript` (plural) to match up with the semantics :) #grammarNazy

Reply:
nice catch thanks!